pxtcompiler/emitter/decompiler.ts

```diff
@@ -343,6 +372,40 @@ ${output}</xml>`;
             result.success = false;
         }
 
+        function countBlock() {
+            emittedBlocks ++;
+            if (emittedBlocks > MAX_BLOCKS) {
+                let e = new Error(Util.lf("Could not decompile because the script is too large"));
```

Comment:
Maybe just "There are too many blocks!"

Reply:
Actually, the user won't see that text. I'll remove the call to lf